packages/odyssey-react/src/components/Table/TableHeader.tsx

```diff
@@ -0,0 +1,35 @@
+/*!
+ * Copyright (c) 2021-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import React from 'react';
+import type { ReactNode, DetailedHTMLProps, HTMLAttributes } from 'react';
+
+export type Props = {
+  children?: ReactNode,
```

Comment:
do we want `children` to be optional? I would think we'd want to enforce the presence of children for components like Header since presumably some amount of column information should be present if you plan to render a table? Perhaps that assumption doesn't hold if we consider tables that have dynamic content?

Reply:
Me feeling is since these components are so low level, if an empty `thead` is valid in HTML it should be here as well